playground/components/header.tsx

```diff
@@ -61,88 +71,109 @@ export const Header: Component<{
               </svg>
-            }
-          >
-            <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-              <path
-                stroke-linecap="round"
-                stroke-linejoin="round"
-                stroke-width="2"
-                d="M12 3v1m0 16v1m9-9h-1M4 12H3m15.364 6.364l-.707-.707M6.343 6.343l-.707-.707m12.728 0l-.707.707M6.343 17.657l-.707.707M16 12a4 4 0 11-8 0 4 4 0 018 0z"
-              />
-            </svg>
-          </Show>
-        </button>
-
-        <label
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 cursor-pointer"
-          title="Import from JSON"
-        >
-          <input type="file" class="sr-only" onChange={uploadFile} accept=".json" />
-          <span class="sr-only">Import from JSON</span>
-          <Icon path={upload} class="h-6" />
-        </label>
-
+              <span class="text-xs md:sr-only">Export to JSON</span>
+            </button>
+
+            <button
+              type="button"
+              onClick={() => exportToCsb(props.tabs)}
+              class="flex flex-row space-x-1 items-center px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
+              title="Export to CodeSandbox"
+            >
+              <svg class="fill-current h-6" style={{ padding: "0px 1.63px 0px 1.63px" }} preserveAspectRatio="xMidYMid" viewBox="0 0 256 296">
```

Comment:
This can be written as follow, although I'm not entirely sure about the `1.63px`:
```suggestion
              <svg class="fill-current h-6 px-[1.63px] py-0" preserveAspectRatio="xMidYMid" viewBox="0 0 256 296">
```